ui-tests/src/test/java/org/openmrs/reference/page/EditReportPage.java

```diff
@@ -0,0 +1,46 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class EditReportPage extends Page {
+	
+   private static final By PREVIEW_BUTTON = By.cssSelector("#previewButton");
+   private static final By START_DATE_FIELD = By.cssSelector("#startDate");
+   private static final By END_DATE_FIELD = By.cssSelector("#endDate");
+   private static final By RUN_BUTTON = By.cssSelector("#preview-parameterizable-button");
+   private static final By CLOSE_TAB = By.xpath("//span[contains(text(),'close')]");
+   private static final String START_DATE = "05/07/2008";
+   private static final String END_DATE= "20/06/2020";
```

Comment:
worth adding a space to separate the variable name with the operator. 

Something like;
```js
private static final String END_DATE = "20/06/2020";
```